server/settings.py

```diff
@@ -107,6 +107,11 @@ class UnrealSettings(BaseSettingsModel):
         enum_resolver=_loaded_asset_enum,
         description="The overriding folder representation type during loading"
     )
+    halon_storage_path: str = SettingsField(
+        "",
+        title="Halon Custom Storage Path",
```

Comment:
I don't think we should label anything within tool with Halon. If by chance we want to merge with work back to Ynputs repos we should use name without Halon